### PR TITLE
bump codecov version

### PR DESCRIPTION
**Pull request by steeling**
Codecov 1.1.0 fixes some panics I've been seeing in some of my dev branches

### CODE REVIEW

**Review by jaellio — APPROVED** [review]

**Review by keithmattix — APPROVED** [review]

File: scripts/test-w-coverage.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ go test -timeout 120s \
    -failfast \
    -race \
    -v \
+   -coverpkg=./pkg/...,./cmd/... \
```
Comment (shashankram) [inline scripts/test-w-coverage.sh:19]: Why is this addition required?
Could you add a sentence in the commit description regarding this for additional context.
